java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -1372,4 +1372,298 @@ CompletableFuture<GlideString[]> xclaimJustId(
      */
     CompletableFuture<Map<GlideString, Object>[]> xinfoConsumers(
             GlideString key, GlideString groupName);
+
+    /**
+     * Transfers ownership of pending stream entries that match the specified criteria.
+     *
+     * @see <a href ="https://valkey.io/commands/xautoclaim">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param group The consumer group name
```

Comment:
Please fix in all your new APIs
```suggestion
     * @param group The consumer group name.
```